src/app/components/algorithms/algorithm-properties/algorithm-properties.component.scss

```diff
@@ -0,0 +1,7 @@
+.algorithm-properties-container{
+  margin: 16px;
```

Comment:
Why not just use bootstrap margins? [See here](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
Removed this css, already used the bootstrap margins on the div just forgot to remove this css